src/pages/en/reference/adapter-reference.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Astro Adapter API (experimental)
+---
+
+Astro is designed to make it easy to deploy to any cloud provider for SSR (server-side rendering). This ability is provided by __adapters__, which are [integrations](./integrations-reference/).
+
+> Server-side rendering in Astro is *experimental*. If you are interested in building an adapter for a host now is the perfect time to help shape these APIs. If you are worried about breaking changes this might be a little too soon for you.
+
+## What is an adapter
+
+An adapter is a special kind of [integrations](./integrations-reference/) that provide an entrypoint for server-side rendering. An adapter does two things:
+
+- Implements host-specific APIs for handling requests.
+- Configures the build according to host conventions.
+
+## Building an Adapter
+
+An adapter is an [integration](./integrations-reference/) and can do anything that an integration can do.
+
+An adapter __must__ call the `setAdapter` API in the `astro:config:done` hook like so:
+
+```js
+export default function createIntegration() {
+  return {
+    name: '@matthewp/my-adapter',
+    hooks: {
+      'astro:config:done': ({ setAdapter }) => {
+        setAdapter({
+          name: '@matthewp/my-adapter',
+          serverEntrypoint: '@matthewp/my-adapter/server.js'
+        });
+      },
+    },
+  };
+}
+```
+
+The object passed into `setAdapter` is defined as:
+
+```ts
+interface AstroAdapter {
+	name: string;
+	serverEntrypoint?: string;
+	exports?: string[];
+}
+```
+
+The properties are:
+
+* __name__: A unique name for your adapter, used for logging.
+* __serverEntrypoint__: The entrypoint for server-side rendering.
+* __exports__: An array of named exports when used in conjunction with `createExports` (explained below).
+
+### Server Entrypoint
+
+Astro's adapter API attempts to work with any type of host, and gives a flexible way to confirm to the host APIs.
```

Comment:
"way to confirm to the host APIs."

(Sorry, I'm not familiar with the proper language to describe this, so I'm just checking that this is intentional wording. In plain English, I'd be expecting "to confirm "X" to the host APIs" or "to provide confirmation to the host APIs")

Reply:
Oops, I meant __conform__

Reply:
Ha! That was my next guess!  :)